Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -128,12 +128,14 @@ + (NSString *)testEnvironmentWithConfiguration:(BPConfiguration *)config {
         BPXCTestFile *xctTestFile = [BPXCTestFile BPXCTestFileFromExecutable:executable isUITestFile:(config.testRunnerAppPath == nil) withError:&error];
         NSAssert(xctTestFile != nil, @"Failed to load testcases from %@", [error localizedDescription]);
         NSMutableSet *testsToSkip = [[NSMutableSet alloc] initWithArray:xctTestFile.allTestCases];
-        NSSet *testsToRun = [[NSSet alloc] initWithArray:config.testCasesToRun];
+        NSSet *testsToRun = [[NSSet alloc] initWithArray:[xctTestFile expandTests:config.testCasesToRun]];
```

Comment:
I checked code base again, maybe it is better to sanitize config file in main.m, before calling `BPRunner *runner = [BPRunner BPRunnerForApp:app withConfig:config withBpPath:nil];`